charms/worker/k8s/uv.lock

```diff
@@ -0,0 +1,619 @@
+version = 1
```

Comment:
Is this auto-generated or is it supposed to be checked in?
I guess checked in is the right thing. I dunno

Reply:
Yes, this must be checked in. This would serve as the source of truth for the pinned dependencies. Also, we can receive autoupdates from dependabot.